### PR TITLE
HPCC-20060 Add es translations

### PR DESCRIPTION
**Pull request by kunalaswani**
Translations added for Spanish.

Signed-off-by: Aswani, Kunal <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by kunalaswani** [comment]
@miguelvazq please review, some of the translations were already present and were not changed, only new translations were added.

**Comment by kunalaswani** [comment]
@rpastrana can you please review the added translations? Thanks

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 1]
```diff
@@ -5,10 +5,15 @@ define(
     AbortedTime: "Abortado a las",
     About: "Acerca",
     AboutGraphControl: "Acerca del controlador gráfico",
+    AboutHPCCSystems: "Acerca de HPCC Systems®",
     AboutHPCCSystemsGraphControl: "Acerca del controlador gráfico de HPCC Systems®",
+    AboutToLoseSessionInformation: "Está a punto de cerrar la sesión y perder toda la información de la sesión. ¿Desea continuar?",
     Account: "Cuenta",
     Action: "Acción",
     Activate: "Activar",
+    ActivateQuery: "Activar Query",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: I would translate Query to "consulta" here in other lines

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 2]
```diff
@@ -192,13 +207,15 @@ define(
...
     ESPNetworkAddress: "Dirección de red de ESP",
+    ESPProcessName: "Nombre del proceso ESP",
     EventName: "Nombre del Evento",
     EventNamePH: "eventname",
     EventScheduler: "Planificador de eventos",
     EventText: "Texto del Evento",
     EventTextPH: "Texto del evento",
     Exception: "Excepción",
     ExpandAll: "Expandir Todo",
+    ExpireDays: "Se vencer en (días)",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: Se vence en

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 3]
```diff
@@ -219,6 +236,7 @@ define(
     FileScopes: "Ámbitos de Archivos",
     FileSize: "Tamaño de archivo",
     FilesNoPackage: "Archivos sin ",
+    FilesWarning: "La cantidad de archivos devueltos es demasiado grande. Solo se devolvieron los primeros 100.000 archivos ordenados por fecha / hora modificada. Si desea limitar los resultados, configure un filtro.",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: What's the original text for  FilesWarning in the english. why  is 100.000 using floating point?

**Review by rpastrana — COMMENTED** [review]
@kunalaswani looks good. A few comments.

File: esp/src/eclwatch/nls/es/hpcc.js (hunk 3 above)
Comment (kunalaswani) [reply esp/src/eclwatch/nls/es/hpcc.js]: The number of files returned is too large. Only the first 100,000 files sorted by date/time modified were returned. If you wish to limit results, set a filter.

**Comment by kunalaswani** [comment]
@rpastrana changes made

**Review by rpastrana — APPROVED** [review]
@kunalaswani

**Pull request merged**